jumanji/environments/routing/robot_warehouse/env.py

```diff
@@ -491,7 +491,8 @@ def reward_and_update_request_queue_if_shelf_in_goal(
             return key, reward, request_queue, shelves
 
         # check if shelf is at goal position and in request queue
-        cond = (shelf_id != 0) & jnp.isin(shelf_id, request_queue + 1)
+        cond = jnp.logical_not(jnp.equal(shelf_id, 0))
+        cond = jnp.logical_and(cond, jnp.isin(shelf_id, request_queue + 1))
```

Comment:
Could we change this to:

`shelf_at_goal = (~jnp.equal(shelf_id, 0)) & jnp.isin(shelf_id, request_queue + 1)`

Makes the `cond` variable more explicit and uses `~` and `&` instead of the jax functions

Reply:
Thanks Sasha, when making these suggestions did you rerun the test? Since the issue fixed here crept in by accepting a similar suggestion, so I'm hesitant to do it again. Similar with the one suggested below. At this stage, since it is so minor, I'm inclined to keep what works. :)

Reply:
I lean towards @sash-a 's, yet both expressions are probably converted to the same XLA statement, so happy to keep as is.